### ISSUE TITLE
Sensors incorrectly marked as "Configuration"

### ISSUE THREAD

**Issue by frenck** [home-assistant/android#2355]
**Home Assistant Android version:** Not sure where to find the version in Sidequest.
Version dates from 19 Feb

**Android version:** ?

**Phone model:** Oculus Quest 2

**Home Assistant version:** 2022.4.0dev0

**Last working Home Assistant release (if known):** Never used Android before... Dunno.

**Description of problem:**

The call volume sensor is listed as "Configuration" entity category. However, since a it is a sensor entity (and thus read only) that is incorrect. Read-only sensors should be either primary OR marked as diagnostic.

I have many sensors disabled, but I suspect this will be the case with other sensors as well.

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**

```

```

**Screenshot of problem:**

<img width="388" alt="image" src="https://user-images.githubusercontent.com/195327/157330825-3deace77-a184-47a5-9a89-23c160cf9473.png">

**Additional information:**

**Comment by frenck** [comment]
Sure.. but it is still incorrectly displayed at this time. I agree if it is a number entity it would be config, but the fact is: it isn't.

Thus a read only sensor is marked as configuration. This is not correct and would also be reviewed as such when used like this in a core PR review.

**Comment by balloob** [comment]
I agree with Frenck. Config entities need to be able to be used to edit the config.

**Comment by dshokouhi** [comment]
Ok will get the sensors updated
